ui/src/AddUserToGroup.js

```diff
@@ -67,16 +77,27 @@ function AddUserToGroup(props) {
     return undefined;
   }
 
+  function isEmailAlreadyInGroup(userInput) {
+    return location.state.existingEmailsInGroup.includes(userInput);
+  }
+
   async function addUserToGroup() {
     if (addUserInProgress) {
       return;
     }
 
     addUserInProgress = true;
 
-    setErrorSummary("");
+    setErrorSummary([]);
     setHasErrors(false);
 
+    if (isEmailAlreadyInGroup(value)) {
+      setErrorSummary(["Email already exists for this group"]);
```

Comment:
Minor wording point, I think this should be phrased as something like `User is already a member of this group`.